src/cloudformation/vpc.template

```diff
@@ -232,37 +246,99 @@
           {
             "Key": "Name",
             "Value": {
-              "Fn::Join": ["-", [{"Ref": "AWS::StackName"}, "private"]]
+              "Fn::Join": ["-", [{"Ref": "AWS::StackName"}, "Private01"]]
             }
           }
         ]
       }
     },
+    "PrivateRouteTable02": {
```

Comment:
I'm wondering if we can do this different, more dynamic.

We have this code to detect which AZ's are alive:
https://github.com/pebble/flotilla/blob/master/src/flotilla/client/region_meta.py#L57

Possible way to do this cross region, would be first to determine which AZ's are available, then having flotilla (like git-deploy.template) create a parent stack, with the N amount of AZ's as nested stacks. Possibly the nested stack could return the EIP value, but it looks like it's not being used for something (yet).